src/pages/monetization/billing-api/index.mdx

```diff
@@ -244,7 +244,7 @@ If you want to use this API, you must create an app in Developer Console and che
 
 ## Endpoints
 
-Base URL: `https://billing.livechatinc.com`
+Base URL: `https://billing.text.com`
 
 - `GET /v2/ledger/livechat` - returns the current ledger. It lists up to 20 entries, use `?page=X` for pagination. Required format: `{result: [LEDGER ENTRY 1, LEDGER ENTRY 2, ...]}`,
```

Comment:
@olek-p should these two ledger endpoints point to v3 API as well?

Reply:
No, I intentionally left `v2` here. Ledger is an abandoned feature and we didn't want to move it to the newer API version.